modules/govuk/templates/node/s_db_admin/reboot_elasticache.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import boto3
+import logging
+
+
+def create_elasticache_client():
+  try:
+    logging.info("getting elasticache client")
```

Comment:
Cronjobs should be silent for normal operation

Reply:
yeah actually it is silent as the default log level is warning. I need to fix some other stuff with the logging though